### ISSUE TITLE
Design Intent - Accessibility Feedback - IIR 1095-B MVP

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#101996]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/gb1dfnkl) at any time._

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- Sorry for the delayed written feedback! We discussed the most important item in Slack already, but if you have any questions about that or any of the other feedback please don't hesitate to reach out.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [x] **Must:** An accessible version of the 1095-B form must be provided. As [discussed in Slack](https://dsva.slack.com/archives/C05RJS5DANT/p1738005741328519), the data in a completed 1095-B isn't announced to screen reader users in context with the field names. Options here would include (any would suffice for an MVP):
   - Determining how to ensure that the 1095-B that's generated is fully accessible
   - Providing an HTML version of the 1095-B (doesn't have to be the same look-and-feel, but should match the PDF in terms of what data is presented)
   - Providing a downloadable TXT version of the 1095-B (again, doesn't have to match the appearance, just the data)
- [x] **Must:** Be careful about heading structure --- making sure the heading hierarchy makes sense for how the information on the page is organized. I'm especially thinking of those alerts-in-cards, if you end up going that route after all. The best exercise for this is to strip away all of the styling and visual design work and just make an outline of hte page, like it's 9th grade English class. If you've got an outline that makes sense, you can map that to your heading hierarchy.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone

**Comment by raquelou** [comment]
@briandeconinck we are going to add a text file. I updated the [figma design](https://www.figma.com/design/cP7JJ9ExBtn2jNax9cfinA/1095-B?node-id=143-6763&t=VbFJ2jUQy9F2fsW2-1) with a link for that. does this Should we include different text in the parenthesis we want to distinguish which is best for printing and which is more accessible. Do you support what we have in the figma file?

**Comment by raquelou** [comment]
@briandeconinck I also would like your thoughts on if the alert that displays when a download fails should be displayed above or below the download links. It is currently placed above the links within the card.

<img width="549" alt="Image" src="https://github.com/user-attachments/assets/577d6747-d8ba-45fc-9642-38e60258c40f" />

**Comment by raquelou** [comment]
We verified with Brian that the heading structure made sense for accessibility.
